Cargo.toml

```diff
@@ -9,5 +9,10 @@ homepage = "https://github.com/conradkleinespel/rustastic-password"
 readme = "README.md"
 keywords = ["read", "password"]
 
-[dependencies]
```

Comment:
To make it build for myself.
Anyway, I have re-added that dependency, but target specific.